packages/components/src/2016-12-01/button.js

```diff
@@ -27,13 +28,75 @@ class Button extends HTMLElement {
     shadowRoot.innerHTML = `
       <orion-inline
         border-radius="2"
-        background="black"
-        color="white"
+        background="white"
+        color="black"
         padding-horizontal="3"
         padding-vertical="2"
-        dim
         pointer><slot /></orion-inline>
     `;
+    this.shadowEl = shadowRoot.children[0];
+
+    this.addEventListener('mouseenter', () => {
+      if (this.state.disabled) { return; }
+      const nextState = ButtonState.enterHover(this.state);
+      this.dispatchEvent(new CustomEvent('change', { detail: nextState }));
```

Comment:
The end user may want to know what caused the change. We have a tradeoff to decide on. We can either dispatch custom events unique to each source event or we can dispatch a single change event with the event source and new state inside the detail. My first impression is to prefer the later:

```
new CustomEvent('change', { detail: { type: 'mouseenter', state: nextState } });
```

Thoughts?